backends/occa/ceed-occa-restrict.okl

```diff
@@ -113,9 +113,25 @@
 }
 
 // *****************************************************************************
-@kernel void kRestrict6(const double* uu,
-                        double* vv) {
-  for (int i=0; i<nelem_x_elemsize_x_ncomp; i++; @tile(TILE_SIZE,@outer,@inner)){
-    vv[i] = uu[i];
+@kernel void kRestrict6(const double* uu, double* vv) {
+  for (int e=0; e<nelem; e++; @tile(TILE_SIZE,@outer,@inner)){
+    for (int k=0; k<numcomp; k++) {
+      for (int n=0; n<elemsize; n++) {
+        vv[n + k*elemsize + e*elemsize*numcomp] =
+          uu[n*stridenode + k*stridecomp + e*strideelem];
+      }
+    }
+  }
+}
+
+// *****************************************************************************
+@kernel void kRestrict7(const double* uu, double* vv) {
+  for (int e=0; e<nelem; e++; @tile(TILE_SIZE,@outer,@inner)){
+    for (int k=0; k<numcomp; k++) {
+      for (int n=0; n<elemsize; n++) {
+        vv[n*stridenode + k*stridecomp + e*strideelem] +=
+          uu[n + k*elemsize + e*elemsize*numcomp];
+      }
+    }
```

Comment:
I didn't try to make this kernel especially fast since we will be replacing the entire OCCA backend at some point.